configs/LLNL-Sierra-IBM-power9-V100-Infiniband/spack.yaml

```diff
@@ -6,12 +6,16 @@
 spack:
   packages:
     default-compiler:
-      spack_spec: xl@16.1.1-2022.08.19-cuda{default_cuda_version}
+      spack_spec: clang@16.0.6-cuda{default_cuda_version}
```

Comment:
xl should be the default compiler on sierra.  Please create compiler-clang to use for Gromacs if you must.